ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml

```diff
@@ -0,0 +1,110 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-gcp-414
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: control-plane-24nodes
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    workflow: openshift-qe-control-plane
+- as: data-path-9nodes
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "9"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
```

Comment:
Replace the pre & post chains with `workflow: openshift-qe-installer-gcp` 
https://steps.ci.openshift.org/workflow/openshift-qe-installer-gcp